runtime/src/accounts_db.rs

```diff
@@ -1133,7 +1158,17 @@ impl AccountsDB {
         rewrite_elapsed.stop();
 
         let mut drop_storage_entries_elapsed = Measure::start("drop_storage_entries_elapsed");
-        drop(dead_storages);
+        let mut recycle_stores = self.recycle_stores.write().unwrap();
```

Comment:
I'm curious about the contention on this lock, so would be nice to factor out the `self.recycle_stores.write().unwrap();` into a separate timing field outside of `Measure::start("drop_storage_entries_elapsed");`

Reply:
added